diff --git a/arm-none-eabi-gcc.ini b/arm-none-eabi-gcc.ini
deleted file mode 100644
index d31183c..0000000
--- a/arm-none-eabi-gcc.ini
+++ /dev/null
@@ -1,3 +0,0 @@
-[built-in options]
-c_args = ['-mthumb', '-mcpu=cortex-m0', '-march=armv6-m']
-       
diff --git a/default.nix b/default.nix
index 5ef5a95..6eab857 100644
--- a/default.nix
+++ b/default.nix
@@ -3,8 +3,6 @@
   stdenv,
   meson,
   ninja,
-
-  mesFlags ? [ "--cross-file=./arm-none-eabi-gcc.ini" ],
 }:
 let
   fs = lib.fileset;
@@ -30,7 +28,5 @@ stdenv.mkDerivation (self: {
     ninja
   ];
 
-  mesonFlags = mesFlags;
-
   hardeningDisable = [ "all" ];
 })
